docker-compose.yml

```diff
@@ -13,23 +13,24 @@ services:
     volumes:
       - UserData:/server/UserData
   db:
-    image: mariadb
+    image: postgres
     restart: always
     ports:
-      - "127.0.0.1:3306:3306"
+      - "127.0.0.1:5438:5432"
     volumes:
-      - MariaDBData:/var/lib/mysql
+      - POSTGRESData:/var/lib/postgresql/data 
     environment:
-      MARIADB_ROOT_PASSWORD: CHANGEME
-      MARIADB_USER: evosc
-      MARIADB_PASSWORD: evosc123!
-      MARIADB_DATABASE: evosc
-      MARIADB_AUTO_UPGRADE: always
-  adminer:
-    image: adminer
+      - POSTGRES_USER=evosc
+      - POSTGRES_PASSWORD=evosc123!
+      - POSTGRES_DB=evosc
+  pgadmin:
+    image: dpage/pgadmin4
     restart: always
+    environment:
+      PGADMIN_DEFAULT_EMAIL: admin@admin.com
+      PGADMIN_DEFAULT_PASSWORD: evosc123!
```

Comment:
Perhaps default password should rather be something like: "CHANGEME"?

Reply:
yes, i'll refer to that in the getting startet guide, that one should _really_ change it